### PR TITLE
Fix crosshair tooptip position

### PR DESCRIPTION
**Pull request by MahboobehMohammadi**

### CODE REVIEW

File: src/store/CrosshairStore.js [hunk 1]
```diff
@@ -316,7 +316,7 @@ class CrosshairStore {
     updateTooltipPosition() {
         this.isArrowLeft = CIQ.ChartEngine.crosshairX <= MAX_TOOLTIP_WIDTH;
         this.left = CIQ.ChartEngine.crosshairX;
-        this.top = CIQ.ChartEngine.crosshairY;
+        this.top = CIQ.ChartEngine.crosshairY - this.stx.top;
```
Comment (brucebinary) [inline src/store/CrosshairStore.js]: @MahboobehMohammadi why is this needed now? The crosshair position looks fine in beta

Comment (MahboobehMohammadi) [reply src/store/CrosshairStore.js]: @brucebinary as you see here crosshair position is not correct when chart is not at the top left of screan since `CIQ.ChartEngine.crosshairX` returns current X screen coordinate of the crosshair.

Comment (MahboobehMohammadi) [reply src/store/CrosshairStore.js]: @brucebinary ?

Comment (brucebinary) [reply src/store/CrosshairStore.js]: @MahboobehMohammadi ops. Lost track of this. Do make a card for this so it can be tracked from the board.

Comment (brucebinary) [reply src/store/CrosshairStore.js]: @MahboobehMohammadi owh ok I saw the PR 🤦‍♂️
